src/app/Dice.elm

```diff
@@ -0,0 +1,19 @@
+module Dice exposing (emptyCup, Cup, cupFromIntList)
```

Comment:
pulled the dice (& cup) stuff into its own module because it feels like it's separate. This lets us change it on its own. If we decide later to change how Cup is implemented, we'd only be changing this one file, more or less. (still need to implement `cupToIntList` to make the interface complete, but we're not using it yet.)